src/pages/Design/DesignTwoUp.vue

```diff
@@ -0,0 +1,50 @@
+<template>
```

Comment:
This looks like a variation you can create with DynamicHeroClassic and various style attributes in the Ui setting DataObject.